app/controllers/api/verify/password_confirm_controller.rb

```diff
@@ -42,15 +42,21 @@ def add_proofing_component(user)
         ProofingComponent.create_or_find_by(user: user).update(verified_at: Time.zone.now)
       end
 
-      def completion_url(result)
-        if result.extra[:profile_pending]
+      def completion_url(result, user)
+        if in_person_profile?(user)
+          idv_in_person_ready_to_verify_url
+        elsif result.extra[:profile_pending]
           idv_come_back_later_url
         elsif current_sp
           sign_up_completed_url
         else
           account_url
         end
       end
+
+      def in_person_profile?(user)
+        ProofingComponent.find_by(user: user)&.document_check == DocAuth::Vendors::USPS
```

Comment:
Alternatively, we could look for an enrollment for the user and use that as the primary indicator. I expect we'll probably want to be tracking the proofing component regardless though.

Reply:
I'm working on a local branch from this one to build out the rest of the content for this page, and we'll most certainly need to reference the actual enrollment record associated with this proofing attempt, so I'm thinking it'd probably be best to consider that value here. That being said, we don't actually create an enrollment yet, so maybe we should revisit that once we are.

@tomas-nava Do we have a ticket which captures the creation of the enrollment record?

Reply:
+1 on using the enrollment. That does seem like a better source of truth

Reply:
Agreed, I think we should check the enrollment. I don't think there's a ticket yet for creating the enrollment

Reply:
Based on my read of LG-6708 / #6556, the new "establishing" status looks like it'll be used so we can create the enrollment sometime early on in the flow. I think once we have that, I'll plan to update to reference the enrollment record. I'll leave some "WILLFIX" comments and can hopefully resolve it in one of the further iterations of the work of this ticket.

Reply:
The new ticket LG-6868 will probably tie into this as well, since we'll need to consider the enrollment as part of updating the profile at the password confirmation step.